### DISCUSSION TITLE
Suggestion: 1.18.2 Wicker Patch usage expansion

### DISCUSSION THREAD

**Discussion by ZephyrWindSpirit** [malcolmriley/minecraft-textiles#64] [Ideas and Suggestions]
Fishing nets that float on the water like the ones from Actual Additions from 1.12.2 or Mariculture 1.7.10?

They slowly generate fish, and a basket could pick it up in vicinity. 

Can wicker baskets create a double wicker basket that gets a lid?

**Comment by malcolmriley** [comment]
I'm not familiar with those mods. I've heard of Actual Additions (I think) but I never played with it.

I do like the idea of using baskets in some kind of baited critter-trap (fishing or frogs or rabbits or whatever). I'll stew this for a bit and see if I can't think of something interesting.

**Comment by malcolmriley** [comment]
What did you have in mind for the lidded-double-baskets? Would they have something that double chests don't already (besides visual)?

**Reply by ZephyrWindSpirit** [reply]
lidded double basket = bigger critter trap. Catch cows or something. Sturdy double lidded baskets could catch hostile mobs?

**Reply by malcolmriley** [reply]
I can see the utility there. The object would have to be picked up and placed as a single item rather than like a double chest I think, like how one bed places two blocks.

I'll keep this suggestion in mind for when I decide to visit the critter-trap-basket feature.
